content/docs/contributing/release-process.md

```diff
@@ -472,7 +471,7 @@ page if a step is missing or if it is outdated.
 
        ```bash
        # Must be run from the cert-manager folder.
-       export GITHUB_TOKEN=*your-token*
+       export GITHUB_TOKEN=$(gh auth token)
```

Comment:
Wow, haven't tried this yet, but might just be the amazing timesaver I was looking for.
I have always done this through the UI, which takes a lot of time IMO.